save-backend/src/main/kotlin/com/saveourtool/save/backend/service/OrganizationService.kt

```diff
@@ -69,7 +69,18 @@ class OrganizationService(
      * @param organization
      * @return organization
      */
-    fun updateOrganization(organization: Organization): Organization = organizationRepository.save(organization)
+    @Suppress("UnsafeCallOnNullableType")
+    fun updateOrganization(organization: Organization): Organization = organizationRepository.findByName(
+        organization.name
+    )!!.let { organizationFromDb ->
+        organizationRepository.save(organizationFromDb.copy(canCreateContests = organization.canCreateContests).apply { id = organizationFromDb.id })
```

Comment:
so `updateOrganization` updates only `canCreateContests`?

Reply:
Oh, my bad.
It should change everything except canCreateContests. Will fix it

Reply:
Done